### PR TITLE
adding ducks, goats and alpacas

### PR DESCRIPTION
**Pull request by Abdyman**
adding duck, goat, alpaca

### CODE REVIEW

**Comment by Abdyman** [comment]
assets PR: https://github.com/TerraFirmaCraft/TFC-Assets/pull/109

**Comment by Abdyman** [comment]
also, currently calling them alpacas but could easily be llamas. just went with that name because from what i saw alpacas are bred more for their wool in comparison to llamas

**Comment by DisasterMoo** [comment]
Please use the code style to reformat your code before we review.

File: src/main/java/net/dries007/tfc/objects/entity/animal/EntityAlpacaWoolTFC.java [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.DifficultyInstance;
+import net.minecraft.world.IBlockAccess;
+import net.minecraft.world.World;
+import net.minecraft.world.storage.loot.LootTableList;
+import net.minecraftforge.common.IShearable;
+import net.minecraftforge.fml.relauncher.Side;
+import net.minecraftforge.fml.relauncher.SideOnly;
+
+public class EntityAlpacaWoolTFC extends EntityAnimal implements IShearable
```
Comment (DisasterMoo) [inline src/main/java/net/dries007/tfc/objects/entity/animal/EntityAlpacaWoolTFC.java]: This one, wasn't supposed to be an extension from the last one?
```java
public class EntityAlpacaWoolTFC extends EntityAlpacaTFC
{
    // code ...
} 
```

Also, the other one seems to be shearable too, duplicate?

Comment (Abdyman) [reply src/main/java/net/dries007/tfc/objects/entity/animal/EntityAlpacaWoolTFC.java]: didnt catch that thank you, new commit should have the resolved along with some others i missed

**Comment by Abdyman** [comment]
code has been imported into PR #528

**Pull request closed without merging**